web/locales/en/messages.ftl

```diff
@@ -13,6 +13,9 @@ email-opt-in-info-sub-with-challenge = Receive emails such as challenge and goal
 email-opt-in-privacy-v2 = By opting in to receive emails you state that you are okay with Mozilla handling this info as explained in Mozilla’s <privacyLink>Privacy Policy</privacyLink>.
 indicates-required = * Indicates required field
 not-available-abbreviation = N/A
+banner-error-slow-1 = Sorry, Common Voice is running slowly. Thanks for your interest.
```

Comment:
Would recommend just "Sorry, Common Voice is running more slowly than usual", without the second part.

Reply:
This is good feedback but I will leave this for now, I added 'Thanks for your interest' based on Hillary's feedback and we could go back and forth on the copy forever.